modules/images-managing-images-enabling-imagestreams-kube.adoc

```diff
@@ -0,0 +1,98 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/managing-images/using-imagestreams-with-kube-resources.adoc
+
+
+[id="images-managing-images-enabling-imagestreams-kube_{context}"]
+= Enabling imagestreams with Kubernetes resources
+
+When using this feature, you can only reference imagestreams that reside in the
```

Comment:
What is _this feature_? Do you need to define the feature so that the topic is free-standing?

Reply:
"using imagestream with k8s resources" is the feature.